src/app/elements/page.tsx

```diff
@@ -1,9 +1,66 @@
-import { Chart } from '@/components/Charts/Chart';
```

Comment:
I removed that import because I was getting an "unused vars" error. Since the comments mentioned that page.tsx is intended for testing purposes, I decided to remove it.